redaxo/src/addons/project/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Ich hatte die Lizenz-Datei im project-Addon bewusst weggelassen, da der Code im Projekt-Addon (zumindest bei uns) in der Regel eben nicht MIT-lizensiert ist, sondern proprietär, projekt-spezifisch.
Jedenfalls möchte ich nicht vorgeben, wie der Code lizensiert sein soll, den man in das Addon legt (und dafür ist das Addon ja gedacht, eigenen Code dort zu platzieren).

Ich würde ein solches Plugin daher auch nicht im Projekt-Addon platzieren. Das Plugin scheint ja ein fertig nutzbares Plugin zu sein, wo man nichts am Code ändert (wenn ich das richtig sehe). Das passt für mich nicht ins Konzept des Projekt-Addons, dass eben dafür da ist, projekt-spezifischen Code dort abzulegen.

Man könnte die Funktionalität des Plugins aber natürlich woanders ablegen. 
Habe mir das Plugin im Detail aber noch nicht angesehen und noch keine Meinung dazu gebildet.

Reply:
Bin auch der Meinung, dass das nicht teil des project-Addons seien sollte. Werde das Plugin gleich mal testen.

Reply:
@gharlan @staabm Ok :)) Dachte das passt hier gut rein und wäre auch nützlich.
Aber ihr habt wohl recht mit Lizenz usw.
Dann einfach den PR rejecten und gut ist ...
Habe das selbst als Plugin eines eigenen Addons
@bloep gerne testen!

Reply:
@aeberhard vielleicht erstmal bei FOR veröffentlichen und so Erfahrungen sammeln? Ein Platz im Core wird sich dann evtl. finden, denke ich

Reply:
@skerbis als eigenes Addon statt als plugin?

Reply:
Klar, warum nicht :wink:

Reply:
guter vorschlag. erstmal als addon und wenn es viele gut gebrauchen können, dann kann man es in den core gerne übernehmen.

um ideen zu reifen sind addons ausserhalb des redaxo cores besser, da man dort schneller updaten/iterieren kann und nicht abhängig ist von redaxo core release zyklen.

Reply:
Gerne als AddOn bei FOR! Das kann ruhig auch als Testwiese für AddOns herhalten, finde ich, denn bei FOR sammeln sich Devs und Nutzer gleichzeitig, während spannende Sachen in privaten Nutzer-Repos nicht die gleiche Aufmerksamkeit bekommen.

Reply:
Okay, baue asap das Addon properties bei FOR

Frage: Muss ich den PR closen?

Reply:
@aeberhard ja